tests/api.v2/model/BasicModel.test.js

```diff
@@ -73,6 +73,25 @@ describe('model/BasicModel', () => {
     expect(mockSqlClient.timeout).toHaveBeenCalledWith(4000);
   });
 
+  it('upsert works correctly', async () => {
+    const mockRowKey = 'mockKey';
+    const newDescription = 'newDescription';
```

Comment:
In general, try to use more realistic mock values. The closer to the real data the better the test. I can ,for example, imagine a bug where we remove the spaces in a string before inserting them and this would work ok.

Reply:
this is a random string, it's not for any table in particular.

Also I didn't understand the spaces removal thing.